src/main/java/org/apache/sysml/hops/AggUnaryOp.java

```diff
@@ -329,8 +329,15 @@ public boolean allowsAllExecTypes()
 
 	@Override
 	protected double computeOutputMemEstimate( long dim1, long dim2, long nnz )
-	{		
-		double sparsity = OptimizerUtils.getSparsity(dim1, dim2, nnz);
+	{
+		double sparsity = -1;
+		if (DMLScript.USE_ACCELERATOR) {
+			// The GPU version (for the time being) only does dense outputs
+			sparsity = 1.0;
+		} else {
+			sparsity = OptimizerUtils.getSparsity(dim1, dim2, nnz);
+		}
```

Comment:
Could you please explain what you mean by "compilation of subsequent operations is not affected"
If I revert my changes, SystemML will not correctly account for the fact that the output on the GPU will be dense.